styles.css

```diff
@@ -0,0 +1,453 @@
+    font-family: "Merriweather", serif;
+    font-weight: 600;
+    font-style: normal;
+}
+h1 {font-size: var(--fs-mobile-h1)}
+h2 {font-size: var(--fs-mobile-h2)}
+h3 {font-size: var(--fs-mobile-h3)}
+body {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+input,
+textarea,
+select,
+button {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+input:focus,
+select:focus,
+textarea:focus {
+    outline: none;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.container-center {
+    display: flex;
+    flex-direction: column;
+    max-width: 1200px;
+    padding: 15px;
+    margin: 0 auto;
+    position: relative;
+}
+.btn-styling{
+    background-color: var(--color-pink);
+    color: white;
+    padding: 12px 24px;
+    border: none;
+    border-radius: 999px;
+    text-transform: uppercase;
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+    cursor: pointer;
+    transition: transform 0.2s ease, box-shadow 0.2s ease;
+}
+.btn-styling:hover{
+        transform: translateY(-1px);
+        box-shadow: 0 12px 20px rgba(0, 0, 0, 0.25);
+}
+.bg-gradient {
+    position: relative;
+    background: linear-gradient(90deg, var(--color-blue) 0%, var(--color-purple) 100%);
+    overflow: hidden;
+}
+
+.bg-gradient::before {
+    content: "";
+    position: absolute;
+    inset: 0;
+    background: url(assets/noise-texture.png) repeat;
+    opacity: 0.5;
+    pointer-events: none;
+    z-index: 0;
+}
+.angle-shape-top {
+    clip-path: polygon(0 0, 100% 30px, 100% 100%, 0 100%);
+}
+.angle-shape-bottom {
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 calc(100% - 30px));
+}
+.angle-shape-both {
+    clip-path: polygon(0 0, 100% 30px, 100% 100%, 0 calc(100% - 30px));
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+.container-center__header {
+    color: var(--color-light-gray);
+}
+.container-center__header::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    width: 85px;
+    height: 85px;
+    background-image: url('assets/squiggle_line_stripe.svg');
+    background-repeat: no-repeat;
+}
+.navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+.navbar ul {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 2.5rem;
+    text-decoration: none;
+}
+.navbar a, .navbar a:visited, .navbar a:active {
+    color: var(--color-light-gray);
+    text-decoration: none;
+}
+.navbar a:hover {
+    text-decoration: underline;
+}
+.header-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    margin: 4rem 0 12rem 0;
```

Comment:
Als je van em/rem gebruik wilt maken (wat heel goed is) is het best practice om de basis waarde te declareren bovenaan bij html {font-size: x px}. Bij default is dit 16px.